### ISSUE TITLE
Overlaying point cloud on corresponding panorama images

### ISSUE THREAD

**Issue by Ingsayyad** [potree/potree#482]
How can I overlay point cloud on panorama images. I am working with static laser scanner. The point cloud from each position of the scanner has panorama image. I want to import it in potree and overlay corresponding point cloud on that.

**Comment by antoinebio** [comment]
you can do it with another nice github project : iTowns
[https://github.com/iTowns/](url)

**Comment by antoinebio** [comment]
[](url)https://github.com/iTowns/

**Comment by joki0924** [comment]
We can implement it ourselves.
It's easy when it's done.

**Comment by Raquel0** [comment]
@joki0924 @antoinebio @Ingsayyad   did you get it?

**Comment by yoyomule** [comment]
Can potree now implement such a function?

**Comment by skinkie** [comment]
Isn't this already done? :) Can be implemented using 

```js
async function run(){
        proj4.defs("RD", "+proj=sterea +lat_0=52.15616055555555 +lon_0=5.38763888888889 +k=0.9999079 +x_0=155000 +y_0=463000 +ellps=bessel +towgs84=565.417,50.3319,465.552,-0.398957,0.343988,-1.8774,4.0725 +units=m +no_defs");
        let transform = proj4("RD", "RD");

        let params = {
                transform: transform
        };      

        Potree.Images360Loader.load("../panoramas/amersfoort", viewer, params).then( images => {
                viewer.scene.add360Images(images);
        });     

        viewer.mapView.showSources(true);
}
```

![overlay](https://user-images.githubusercontent.com/502394/92874010-ec063680-f407-11ea-903e-3405a229f6ad.jpg)

**Comment by Lurrechaga** [comment]
@skinkie Most probably you have moved on, is this still working with the ongoing version?. For some reason I cant get this thing working with custom coordinates. This [coment ](https://github.com/potree/potree/commit/b845e49bdabb0a455df56bc792eed0062cf88449#commitcomment-37472780)by the creator says its as simple as that.
But I keep getting this.ToMap.forward is undefined. ToMap does not get defined. directories are right and image360loader gets all the data from the coordinates.txt correctly. 

edit: you code does not work neither.

**Comment by skinkie** [comment]
@Lurrechaga it is still somewhere in my heart. Doesn't it work with more recent versions? Or do you have the same issue with the version of 2020?

**Comment by Lurrechaga** [comment]
@skinkie yup, thats something that didnt cross my mind at all for some reason, tried version 1.7.2 from 2020 and worked perfectly. Thanks lmao

**Comment by skinkie** [comment]
@Lurrechaga maybe we can work collaboratively on getting it to work with the current version?

**Comment by MattiasTancred** [comment]
@skinkie  -- > would it be possible for you to share an example that you know to work, 1 image, pointcloud, coordinates.txt file.
We are struggeling to understand the image importer. We have all the parameters/images etc etc from our ROS system but failing to align in Potree. A simple example with all the components would help us to reverse engineering.

**Comment by skinkie** [comment]
@MattiasTancred like mentioned before. I know that the example with the potree code of 2020. Are you using that, or something more recent?

**Comment by MattiasTancred** [comment]
@skinkie - thats fine, at the moment it does not matter what version we get, just as long as we get a full (image/pointcloud/coordinates.txt) working example we can figure out the rest. If you have a full working example to share we would be grateful.

**Comment by skinkie** [comment]
> @skinkie -- > would it be possible for you to share an example that you know to work, 1 image, pointcloud, coordinates.txt file. We are struggeling to understand the image importer. We have all the parameters/images etc etc from our ROS system but failing to align in Potree. A simple example with all the components would help us to reverse engineering.

Let me know once you have enough info.
http://prorail.opengeo.nl/VST.html

**Comment by MattiasTancred** [comment]
@skinkie -- perfect, thank you very much, I think we have it all.

**Comment by skinkie** [comment]
@MattiasTancred please notice that the projection used here is the Dutch National Grid (RD Amersfoort) which is 'flat'. This dataset was very well curated, this might not be your average input.

**Comment by MattiasTancred** [comment]
@skinkie I forgot to ask, is it possible to get the pointcloud in .las/e57/non-converted format. We are using RVIZ for troubleshooting so we need another format that works.

**Comment by skinkie** [comment]
@MattiasTancred can you send me an email at stefan at opengeo dot nl?

**Comment by wenjing0213** [comment]
@skinkie would it be possible for you to share what are the requirements for capturing panoramic images that can be perfectly superimposed with point clouds? We directly assigned the IMU data in the acquisition process to the panoramic data through the timestamp, but there was an error in this way, so that the panoramic data and the point cloud data could not completely coincide.

**Comment by skinkie** [comment]
@wenjing0213 this is completely offtopic here.

**Comment by wenjing0213** [comment]
@skinkie
I got the following data:

Point cloud 
360 Panoramic images
Reference file for images

Unfortunately, there seems to be a deviation between Point Cloud and 360 Images. I don't know what causes this :( 
 I wonder if it's due to errors in the collection of data

**Comment by skinkie** [comment]
@wenjing0213 offtopic. Please ask such questions on a photogrammetry forum.

**Comment by wenjing0213** [comment]
thank you！

**Comment by BaranMohammadzadeh** [comment]
> @skinkie - thats fine, at the moment it does not matter what version we get, just as long as we get a full (image/pointcloud/coordinates.txt) working example we can figure out the rest. If you have a full working example to share we would be grateful.

@skinkie Hello,
I'd like to kindly request access to the sample data, especially since it seems to reference dates from a few years ago.
Could you also provide the requirements for capturing panoramic images that can be accurately aligned with point clouds? we just need a complete working example (including images, point clouds, and coordinates.txt). We'd really appreciate it if you could share a full working example with us.

**Comment by wenjing0213** [comment]
已收到您的邮件,祝生活愉快~

**Comment by BaranMohammadzadeh** [comment]
@skinkie Hello,
I'd like to kindly request access to the sample data, especially since it seems to reference dates from a few years ago.
Could you also provide the requirements for capturing panoramic images that can be accurately aligned with point clouds? we just need a complete working example (including images, point clouds, and coordinates.txt). We'd really appreciate it if you could share a full working example with us.

**Comment by skinkie** [comment]
@BaranMohammadzadeh offtopic. Please ask such questions on a photogrammetry forum.
